classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialise(label)
```

Comment:
- [ Required ] I think there is a typo on the constructor of this class, the name of the constructor should be `initialize` not `initialise` kindly rectify this.